### DISCUSSION TITLE
31/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#200] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer estuve en la clase del ispc. Tuve otra clase aparte. A la noche ví que quedaba para terminar y nos organizamos en el equipo para juntarnos hoy a las 20hs.
Hoy voy a reunirme con el equipo y tratar de terminar lo que más pueda lo q me queda por hacer.
El tiempo ⏱️ es y sigue siendo mi impedimento jajaja me faltan más hs en el calendario 📆

**Comment by LuciaL152** [comment]
Ayer estuve en las dos clases , además estuve revisando que es lo que estaría faltando, hoy reunión con el equipo.

**Comment by fmdcba** [comment]
Hoy estuve en la reunión con el grupo y tomando nota de lo que falta por hacer. Durante el finde voy a terminar con el video para la presentación final y agregar algunas pruebas de la app móvil al pdf.
